### PR TITLE
clean up config envs

### PR DESCRIPTION
**Pull request by wolverineks**
move feature flags from `config.ts`  to `feature.ts`
clean up imports

### CODE REVIEW

File: src/features.ts [hunk 1]
```diff
@@ -11,4 +8,33 @@ export default {
     search: __DEV__ || false,
     nfts: __DEV__ || false,
   },
+  useTestnet: __DEV__ ? false : false,
```
Comment (stackchain) [inline src/features.ts:11]: for folders `useX` is ok, for vars maybe it can cause some lint issues when importing within react component lets see.

**Review by stackchain — APPROVED** [review]

File: src/features.ts [hunk 2]
```diff
@@ -11,4 +8,33 @@ export default {
...
+  startWithIndexScreen: __DEV__ ? false : false,
+  prefillWalletInfo: __DEV__ ? false : false,
+  showProdPoolsInDev: __DEV__ ? false : false,
+  showNftGallery: __DEV__ ? true : false,
+  moderatingNftsEnabled: __DEV__ ? false : false,
```
Comment (michalszorad) [inline src/features.ts:16]: Let's say someone wants to override these values for local development, and the only way to do it is to modify this file tracked by git. When creating a commit, the person needs to remember not to commit this file. Will it not add extra mental overhead?

File: e2e/createWallet.spec.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // @flow
```
Comment (banklesss) [inline e2e/createWallet.spec.js]:
```suggestion
```

**Review by michalszorad — APPROVED** [review]

File: src/features.ts (hunk 2 above)
Comment (stackchain) [reply src/features.ts:16]: I'm ok with that, as long as the PROD is always in the elses' case :D

Comment (stackchain) [reply src/features.ts:16]: working with feature branches will make this concern go away, also when moving to prod the flag will be gone (at least it's supposed to)

Comment (wolverineks) [reply src/features.ts:16]: gitignore features?

**Pull request merged**